src/assets/v1.3.0/compatibility_fallback_handler.json

```diff
@@ -51,6 +51,7 @@
     "336": "0xf48f2B2d2a534e402487b3ee7C18c33Aec0Fe5e4",
     "338": "0x017062a1dE2FE6b99BE3d9d37841FeD19F573804",
     "420": "0x017062a1dE2FE6b99BE3d9d37841FeD19F573804",
+    "424": "0x1AC114C2099aFAf5261731655Dc6c306bFcd4Dbd",
```

Comment:
this address doesn't seem to match the expected address (0x017062a1dE2FE6b99BE3d9d37841FeD19F573804 or 0xf48f2B2d2a534e402487b3ee7C18c33Aec0Fe5e4)